examples/k8petstore/k8petstore.sh

```diff
@@ -46,173 +46,193 @@ function create {
-  "desiredState": {
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "fectrl",
+    "labels": {"name": "frontend"}
+  },
+  "spec": {
     "replicas": $FE,
-    "replicaSelector": {"name": "frontend"},
-    "podTemplate": {
-      "desiredState": {
-         "manifest": {
-           "version": "v1beta1",
-           "id": "frontendCcontroller",
-           "containers": [{
-             "name": "frontend-go-restapi",
-             "image": "jayunit100/k8-petstore-web-server:$VERSION"
+    "selector": {"name": "frontend"},
+    "template": {
+      "spec": {
+         "containers": [{
+           "name": "frontend-go-restapi",
+           "image": "jayunit100/k8-petstore-web-server:$VERSION"
          }]
-         }
        },
-       "labels": {
-         "name": "frontend",
-         "uses": "redis-master"
+       "metadata": {
+         "labels": {
+           "name": "frontend",
+           "uses": "redis-master"
+         }
        }
-      }},
-  "labels": {"name": "frontend"}
+    }
+  }
 }
 EOF
 
 cat << EOF > bps-load-gen-rc.json
 {
-  "id": "bpsloadgenrc",
   "kind": "ReplicationController",
-  "apiVersion": "v1beta1",
-  "desiredState": {
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "bpsloadgenrc",
+    "labels": {"name": "bpsLoadGenController"}
+  },
+  "spec": {
     "replicas": $LG,
-    "replicaSelector": {"name": "bps"},
-    "podTemplate": {
-      "desiredState": {
-         "manifest": {
-           "version": "v1beta1",
-           "id": "bpsLoadGenController",
-           "containers": [{
-             "name": "bps",
-             "image": "jayunit100/bigpetstore-load-generator",
-             "command": ["sh","-c","/opt/PetStoreLoadGenerator-1.0/bin/PetStoreLoadGenerator http://\$FRONTEND_SERVICE_HOST:3000/rpush/k8petstore/ 4 4 1000 123"]
-         }]
-         }
-       },
-       "labels": {
-         "name": "bps",
-         "uses": "frontend"
+    "selector": {"name": "bps"},
+    "template": {
+      "metadata": {
+        "name": "bpsLoadGenController",
+        "labels": {
+          "name": "bps",
+          "uses": "frontend"
         }
-      }},
-  "labels": {"name": "bpsLoadGenController"}
+      },
+      "spec": {
+        "containers": [{
+           "name": "bps",
+           "image": "jayunit100/bigpetstore-load-generator",
+           "command": ["sh","-c","/opt/PetStoreLoadGenerator-1.0/bin/PetStoreLoadGenerator http://\$FRONTEND_SERVICE_HOST:3000/rpush/k8petstore/ 4 4 1000 123"]
+        }]
+      }
+    }
+  }
 }
 EOF
 
 cat << EOF > fe-s.json
 {
-  "id": "frontend",
   "kind": "Service",
-  "apiVersion": "v1beta1",
-  "port": 3000,
-  "containerPort": 3000,
-  "publicIPs":["$PUBLIC_IP","10.1.4.89"],
-  "selector": {
-    "name": "frontend"
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "frontend",
+    "labels": {
+      "name": "frontend"
+    }
   },
-  "labels": {
-    "name": "frontend"
+  "spec": {
+    "ports": [{
+      "port": 3000,
+      "targetPort": 3000
```

Comment:
The default value of "targetPort" is the same as "port", https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L1108

Please omit the default value.